modules/persistent-storage-csi-manila-install-driver.adoc

```diff
@@ -0,0 +1,89 @@
+* Access to the {product-title} web console and command-line interface (CLI).
+
+.Procedure
+
+To install the Manila CSI driver, complete the following steps:
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Locate the *Manila CSI Driver Operator* from the list and click on the Operator link.
+
+. Create the driver:
+.. From the *Details* tab, click *Create Instance*.
+
+.. Optional: Select *YAML view* to make modifications, such as adding notations, to the ManilaDriver object template.
+
+.. Click *Create* to finalize.
++
+[IMPORTANT]
+====
+Renaming the cluster and specifying a certain namespace are not supported functions.
+====
+
+. From the CLI, create the ManilaDriver object in your {product-title} cluster by specifying the file you  created in the previous step:
++
+----
+$ oc create -f maniladriver.yaml
+----
+
+When finished, the Manila CSI driver is deployed on {product-title} for dynamic provisioning of RWX persistent volumes on OpenStack.
+
+.Verification steps
+. Verify that the ManilaDriver resource was created successfully by entering the following command:
++
+----
+$ oc get all -n openshift-manila-csi-driver
+----
++
+Example output:
++
+----
+NAME                                                              READY   STATUS    RESTARTS   AGE
+pod/csi-nodeplugin-nfsplugin-lzvpm                                1/1     Running   0          18h
+pod/csi-nodeplugin-nfsplugin-slvg2                                1/1     Running   0          18h
+pod/csi-nodeplugin-nfsplugin-xmps9                                1/1     Running   0          18h
+pod/openstack-manila-csi-controllerplugin-7d4f5d985b-mw4x5        3/3     Running   0          17h
+pod/openstack-manila-csi-nodeplugin-6xchs                         2/2     Running   0          18h
+pod/openstack-manila-csi-nodeplugin-bkcmz                         2/2     Running   0          18h
+pod/openstack-manila-csi-nodeplugin-rlpps                         2/2     Running   0          18h
+
+NAME                                                              DESIRED CURRENT READY UP-TO-DATE AVAILABLE NODE SELECTOR AGE
+daemonset.apps/csi-nodeplugin-nfsplugin                           3       3       3     3          3                       18h
+daemonset.apps/openstack-manila-csi-nodeplugin                    3       3       3     3          3                       18h
+
+NAME                                                              READY UP-TO-DATE AVAILABLE AGE
+deployment.apps/openstack-manila-csi-controllerplugin             1/1   1          1         18h
+
+NAME                                                              DESIRED CURRENT READY AGE
+replicaset.apps/openstack-manila-csi-controllerplugin-7d4f5d985b  1       1       1     17h
+
+----
+
+. Verify that the storage class was created successfully by entering the following command:
++
+----
+$ oc get storageclass csi-manila-ceph
```

Comment:
One notice here... StorageClass names can be different, depending on Manila service configuration. 

For instance, in PSI we have just one Manila share type - `ceph`. It means that the operator creates a StorageClass `csi-manila-ceph`. In the upstream they use `nfs`, therefore their StorageClass is called `csi-manila-nfs`: https://github.com/kubernetes/cloud-provider-openstack/blob/master/examples/manila-csi-plugin/nfs/dynamic-provisioning/storageclass.yaml#L4 
And of course there can be several `csi-manila-` StorageClasses, one for each Manila share type.

I think it should be like:

> . Verify that storage classes were created successfully by entering the following command:

and 

> $ oc get storageclasses | grep -E "NAME|csi-manila-"